torchvision/utils.py

```diff
@@ -121,10 +126,60 @@ def save_image(
             If a file object was used instead of a filename, this parameter should always be used.
         **kwargs: Other arguments are documented in ``make_grid``.
     """
-    from PIL import Image
     grid = make_grid(tensor, nrow=nrow, padding=padding, pad_value=pad_value,
                      normalize=normalize, range=range, scale_each=scale_each)
     # Add 0.5 after unnormalizing to [0, 255] to round to nearest integer
     ndarr = grid.mul(255).add_(0.5).clamp_(0, 255).permute(1, 2, 0).to('cpu', torch.uint8).numpy()
     im = Image.fromarray(ndarr)
     im.save(fp, format=format)
+
+
+def draw_bounding_boxes(
+    image: torch.Tensor,
+    boxes: torch.Tensor,
+    colors: Optional[List[str]] = None,
```

Comment:
while having a `str` defining the color of the boxes is probably the simplest in general for the user, given that PIL supports `Tuple[int, int, int]` as well (which is more generic), I wonder if the default annotation and documentation should instead mention the RGB tuple of values instead.

Given that we are not considering torchscript here, we could probably mention that this is a `Union` of those two types.

Thoughts?

Reply:
Sounds good. Done.